flutter_cache_manager/pubspec.yaml

```
@@ -1,27 +1,27 @@
 name: flutter_cache_manager
 description: Generic cache manager for flutter. Saves web files on the storages of the device and saves the cache info using sqflite.
-version: 3.3.0
+version: 3.4.0
 homepage: https://github.com/Baseflow/flutter_cache_manager/tree/master/flutter_cache_manager
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: '>=3.0.0 <4.0.0'
 
 dependencies:
-  clock: ^1.1.0
-  collection: ^1.15.0
-  file: ">=6.0.0 <7.0.0"
+  clock: ^1.1.1
+  collection: ^1.17.0
+  file: '>=6.0.0 <8.0.0'
   flutter:
     sdk: flutter
-  http: ^0.13.0
+  http: '>=0.13.0 <2.0.0'
   path: ^1.8.0
-  path_provider: ^2.0.0
-  rxdart: ">=0.26.0 <0.28.0"
-  sqflite: ^2.0.0+3
+  path_provider: ^2.0.15
+  rxdart: '>=0.26.0 <1.0.0'
+  sqflite: ^2.2.0
```

I don't see a reason to change these. Btw, RxDart 0.28.0 would be a breaking change.
```suggestion
  path_provider: ^2.0.0
  rxdart: '>=0.26.0 <0.28.0'
  sqflite: ^2.0.0+3
```